backend/python/app/services/implementations/file_storage_service.py

```diff
@@ -0,0 +1,73 @@
+import firebase_admin
+from datetime import datetime, timedelta
+from firebase_admin import storage
+
+from ..interfaces.file_storage_service import IFileStorageService
+
+
+class FileStorageService(IFileStorageService):
+    """
+    FileStorageService interface for handling blob storage
+    """
+
+    def __init__(self, logger):
+        """
+        Create an instance of FileStorageService with default bucket
+
+        :param logger: application's logger instance
+        :type logger: logger
+        """
+        self.logger = logger
+        self.bucket = storage.bucket()
+
+    def get_file(self, file_name, expiration_time=timedelta(minutes=60)):
+        blob = self.bucket.get_blob(file_name)
+        if not blob:
+            return None
+        expiration = datetime.now() + expiration_time
+        url = blob.generate_signed_url(expiration)
+        return url
+
+    def create_file(self, file_name, file):
```

Comment:
Can we add a `content_type` parameter for callers to specify MIME type (e.g. `application/pdf`)? Otherwise it will default to `application/octet-stream` in Firebase Storage which is "arbitrary binary data".

cc @emilyslouie @tamim-khan